test/index.js

```diff
@@ -16,1680 +15,1339 @@ const internals = {};
 // Test shortcuts
 
 const lab = exports.lab = Lab.script();
-const describe = lab.describe;
-const it = lab.it;
-const expect = Code.expect;
+const { describe, it, expect } = lab;
+
+
+// Helpers
+const Helpers = require('./helpers');
 
 
 describe('scheme', () => {
 
-    it('fails with no plugin options', (done) => {
+    it('fails with no plugin options', () => {
 
         const server = new Hapi.Server();
-        server.connection();
-        server.register(require('../'), (err) => {
+        server.register(require('../'));
```

Comment:
Shouldn't you `await` on `server.register` as per the new [docs](https://github.com/hapijs/hapi/blob/master/API.md#server.register())

Reply:
yes, you should